apps/daimo-mobile/src/logic/note.ts

```diff
@@ -1,19 +1,30 @@
-import { getNoteClaimSignature } from "@daimo/common";
+import {
+  getNoteClaimSignature,
+  getNoteClaimSignatureFromSeed,
+} from "@daimo/common";
 import { useEffect, useState } from "react";
 import { Address, Hex } from "viem";
 
 export function useEphemeralSignature(
```

Comment:
i'd add a 1 line comment explaining what this does & in particular ephemeralPrivateKey vs seed

Reply:
done